tests/ibge/test_uf.py

```diff
@@ -1,6 +1,9 @@
 from unittest import TestCase
 
-from brutils.ibge.uf import convert_code_to_uf
+from brutils.ibge.uf import (
+  convert_code_to_uf, 
+  convert_uf_to_text,
```

Comment:
Arrumando erro de lint, erro de indentação:

```suggestion
    convert_code_to_uf, 
    convert_uf_to_text,
```